vendor/github.com/kubev2v/lib-volume-populator/populator-machinery/controller.go

```diff
@@ -716,13 +726,27 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 	return nil
 }
 
-func makePopulatePodSpec(pvcPrimeName string) corev1.PodSpec {
+func makePopulatePodSpec(pvcPrimeName, secretName string) corev1.PodSpec {
+	nonRoot := true
+	allowPrivilageEscalation := false
+	user := int64(1000)
 	return corev1.PodSpec{
-		ServiceAccountName: "forklift-controller",
+		//ServiceAccountName: "forklift-controller",
```

Comment:
drop?

Reply:
eh it will require another push to the lib. forgot it, sorry.